src/__tests__/generateConfluenceMarkdown.test.ts

```diff
@@ -0,0 +1,121 @@
+        const ageInDays = now.diff(createdAt, 'days').days;
+        const ageInWholeDays = Math.round(ageInDays);
+
+        const oneDependency = mapWithoutIssues.set(dependency, noIssues);
+        const results = generateConfluenceMarkdown(oneDependency);
+        expect(results).not.toContain(`h3. ${dependency}`);
+        expect(results).not.toContain('\n ||*Title*||*Age*||');
+        expect(results).not.toContain(`|[${fakeIssues[0].title}|${fakeIssues[0].url}]|`);
+        expect(results).not.toContain(`|[${fakeIssues[1].title}|${fakeIssues[1].url}]|`);
+        expect(results).not.toContain(`|${ageInWholeDays}&nbsp;days|`);
+    });
+
+    it('should return two issues for a single dependency with correct markdown', () => {
+        const mockDependencyMap: Map<string, Issue[]> = new Map();
+        const dependency = 'NodeJsDependency';
+
+        const twoIssues = mockDependencyMap.set(dependency, fakeIssues);
+        const results = generateConfluenceMarkdown(twoIssues);
+        expect(results).toContain(`h3. ${dependency}`);
+        expect(results).toContain(`|[${fakeIssues[0].title}|${fakeIssues[0].url}]|`);
+        expect(results).toContain(`|[${fakeIssues[1].title}|${fakeIssues[1].url}]|`);
+    });
+    it('should return two dependencies, with correct markdown for their issues', () => {
+        const mockDependencyMap: Map<string, Issue[]> = new Map();
+        const dependency1 = 'Graphql';
+        const dependency2 = 'TypeStrong';
+
+        mockDependencyMap.set(dependency1, fakeIssues);
+        mockDependencyMap.set(dependency2, singleIssue);
+
+        const results = generateConfluenceMarkdown(mockDependencyMap);
+        expect(results).toContain(`h3. ${dependency1}`);
+        expect(results).toContain(`|[${fakeIssues[0].title}|${fakeIssues[0].url}]|`);
+        expect(results).toContain(`|[${fakeIssues[1].title}|${fakeIssues[1].url}]|`);
+
+        expect(results).toContain(`h3. ${dependency2}`);
+        expect(results).toContain(`|[${singleIssue[0].title}|${singleIssue[0].url}]|`);
+    });
+    it('should return an issue that has square brackets and curly braces in its title', () => {
+        const mockDependencyMap: Map<string, Issue[]> = new Map();
+        const dependency = 'React';
+        singleIssue[0].title = '[Navigation Editor] Dropdown menus too narrow {}';
+
+        mockDependencyMap.set(dependency, singleIssue);
+        const results = generateConfluenceMarkdown(mockDependencyMap);
+        expect(results).toContain(`|[${singleIssue[0].title}|${singleIssue[0].url}]|`);
+        // array.replace(/{|}/g, ''); // TODO..
+    });
+    it('should return a dependency with one issue with all the correct fields and markdown', () => {
+        const mockDependencyMap: Map<string, Issue[]> = new Map();
+        const dependency = 'OSS';
+        singleIssue[0].title = 'Added more info to readme.md';
+
+        const now = DateTime.local();
```

Comment:
We should probably extract a function (at the top of this file) that can return the age in whole days for an iso string that gets passed in. (Avoiding duplicate code)